### PR TITLE
Rename typedef back to previous name for backwards compatibility

### PR DESCRIPTION
**Pull request by dixonjoel**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

#349 renamed MeasurementLinkServerUserEvents to MeasurementLinkServerUserSettings. However, some versions of the Run Service.vi in our template and in instantiated measurement services were wiring a control to this input. So those would be broken by this change.

I'm renaming it back to the old name and removing this control where it was wired in examples and in our template so it can change in the future without backwards compatibility concerns.

### Why should this Pull Request be merged?

Examples and the template would be broken without this change.

### What testing has been done?

Fixed broken Run Service.vi - just ensured they weren't broken.

### CODE REVIEW

File: Source/Example Measurements/NI-Digital SPI/NI-Digital Measurement/Run Service.vi
Comment (dixonjoel) [inline Source/Example Measurements/NI-Digital SPI/NI-Digital Measurement/Run Service.vi:1]: Same diff as NI-DMM

File: Source/Example Measurements/NI-FGEN Standard Function/NIFgenStandardFunction/Run Service.vi
Comment (dixonjoel) [inline Source/Example Measurements/NI-FGEN Standard Function/NIFgenStandardFunction/Run Service.vi:1]: Same diff as NI-Digital and NI-DMM

File: Source/Example Measurements/NI-SCOPE Acquire Waveform/NI-SCOPE Acquire Waveform/Run Service.vi
Comment (dixonjoel) [inline Source/Example Measurements/NI-SCOPE Acquire Waveform/NI-SCOPE Acquire Waveform/Run Service.vi:1]: Same diff as NI-Digital, NI-DMM, and NI-FGEN

File: Source/Generator/MeasurementLink Measurement Template/Run Service.vi
Comment (dixonjoel) [inline Source/Generator/MeasurementLink Measurement Template/Run Service.vi:1]: image

File: Source/Runtime/MeasurementLink Measurement Server/Run Service.vi
Comment (dixonjoel) [inline Source/Runtime/MeasurementLink Measurement Server/Run Service.vi:1]: Relink to typedef.

**Review by pbirkhol-ni — APPROVED** [review]

**Pull request merged**